pandas/core/generic.py

```diff
@@ -5732,59 +5732,65 @@ def clip_lower(self, threshold, axis=None, inplace=False):
 
             .. versionadded:: 0.21.0
 
+        See Also
+        --------
+        DataFrame.clip : General purpose method to trim `DataFrame` values to
+            given threshold(s)
+        DataFrame.clip_upper : Trim `DataFrame` values above given
+            threshold(s)
+        Series.clip : General purpose method to trim `Series` values to given
+            threshold(s)
+        Series.clip_upper : Trim `Series` values above given threshold(s)
```

Comment:
Ideally, for `DataFrame.clip_lower` we would like to have in `See Also`:
- `Series.clip_lower`
- `DataFrame.clip`
- `DataFrame.clip_upper`

So, the same in `Series` and nothing else from `Series`. And the similar from the same class (`DataFrame`). As this is being reused by both `Series` and `DataFrame`, I'd probably list all `clip`, `clip_lower` and `clip_upper`, but I'm not sure. @jreback ?

Reply:
Only caveat here is that the user is working on a doc that is shared by `Series` and `DataFrame`. Mentioned earlier that without further updates (i.e. implementing substitution) that the `clip_lower` reference would be self-referential half of the time. 

I advised just removing it to not get hung up on it for now, the other option would be to explicitly list both `DataFrame.clip_lower` and `Series.clip_lower` and live with the fact that one will be self referencing. No strong preference on my end.

Reply:
Sorry, I was waiting for a little clarification, since I'm not sure how to proceed. I can go either way, but I haven't seen a conclusive definition. Was a decision made on how to proceed with these?

Reply:
If I'm not wrong, in another ticket the final implementation just listed everything. As @WillAyd says in some cases the same method being documented will be self-referenced, but it's not a big issue.